### DISCUSSION TITLE
Add a basic chart or note section

### DISCUSSION THREAD

**Discussion by nfg42** [mealie-recipes/mealie#1413] [Feature Request]
**Before submitting this feature request I have**
_COMPLETE AND FILL THESE OUT_
- [X] Reviewed the documentation and verified that my feature request isn't already a feature.
- [X] Reviewed the existing feature requests and my feature has not been requested.
- [X] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

**Please Describe The Problem To Be Solved**
It would be nice to have a chart or note section that can be referenced from recipes and cookbooks. 
So for example: It would be nice to make a time and temp chart for sous vide for different items and to 
be able to link to it from all sous vide recipes. This could also be used for ingredient substitutions lists, 
measurement conversions, and countless other “I’m stupid and don’t remember stuff” lists.

**Suggested Solution**
* Make a new notes section that is basically just a recipe with a single instruction and none of the other formatting. 
* Maybe keep tags too? Then when you click on a tag for a recipe, it would also display notes with the same tag. 
* I might just use a zero ingredient recipe for this stuff but it would be nice if it had its own menu option and not take up space on recipe lists.
* The only issue I could see for this would be that it might be hard to not make it a full wiki. I definitely want to keep it simple.

**Additional Information**
_FILL THESE OUT_
- [ ] If the feature is accepted I'm willing to submit a PR to provide this feature

**Comment by michael-genson** [comment]
This could be achieved using notes on individual recipes. While not as robust as what you're asking, I think it's sufficient.
